{{ cookiecutter.project_name }}/Dockerfiles/Dockerfile_python3

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
```

Comment:
Even for these `python-pip python-dev build-essential` you need versions. Otherwise things might change and go wrong.

Reply:
pip is getting updated automatically using the line belowI think

Reply:
I am also not sure if we want to install python-dev by default..